lib/plugins/aws/deploy/lib/mergeIamTemplates.js

```diff
@@ -36,16 +36,76 @@ module.exports = {
         .Properties
         .PolicyName = `${this.options.stage}-${this.serverless.service.service}-lambda`;
 
-      iamPolicyLambdaExecutionTemplate
-        .IamPolicyLambdaExecution
-        .Properties
-        .PolicyDocument
-        .Statement[0]
-        .Resource = `arn:aws:logs:${this.options.region}:*:*`;
-
       _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources,
         iamPolicyLambdaExecutionTemplate);
 
+      if (!this.serverless.service.provider.cf_logs) {
```

Comment:
I would like to change this to `cfLogs` or `cloudFormationLogs` or something similar (`camelCase`) as we're using this type of convention throughout the whole `serverless.yml` file and this would be the only exception.